app/src/main/java/net/gsantner/markor/format/markdown/MarkdownTextConverter.java

```diff
@@ -160,26 +159,20 @@ public class MarkdownTextConverter extends TextConverterBase {
     @Override
     public String convertMarkup(String markup, Context context, boolean lightMode, boolean lineNum, File file) {
         String converted = "", onLoadJs = "", head = "";
-
         final MutableDataSet options = new MutableDataSet();
 
-        if (lineNum) {
-            // Add code blocks Line numbers extension
-            final ArrayList<Extension> extensions = new ArrayList<>(flexmarkExtensions);
-            extensions.add(LineNumbersExtension.create());
```

Comment:
So basically this PR removes everything from Java side AST line number parsing and instead line numbers are fully generated at view-time by prism javascript. Right?

Reply:
The work of the extension `flexmark-ext-prism-linenumber` is just to add class `line-numbers` for each code block (`pre`), but I noticed that only add class `line-numbers` to the `body` can also globally enable line numbers, so I removed the extension.

> [Line Numbers ▲ Prism plugins (prismjs.com)](https://prismjs.com/plugins/line-numbers/)